Backend/dev/migrations/provider-dashboard/0032-generic-scheduler-trigger.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO atlas_bpp_dashboard.access_matrix (id, role_id, api_entity, user_access_type, created_at, updated_at, user_action_type) VALUES
+    ('aef13d16-8601-1d2c-084d-eerc1f476a24', '37947162-3b5d-4ed6-bcac-08841be1534d', 'DRIVERS', 'USER_NO_ACCESS', now(), now(), 'SCHEDULER_TRIGGER');
```

Comment:
`insert into atlas_bap_dashboard.access_matrix (id, role_id, api_entity, user_access_type, created_at, updated_at, user_action_type) (select atlas_bap_dashboard.uuid_generate_v4() as id,id as role_id,'CUSTOMERS' as api_entity, 'USER_NO_ACCESS' as user_access_type, now() as created_at, now() as updated_at, 'CUSTOMER_CANCELLATION_DUES_SYNC' as user_action_type from atlas_bap_dashboard.role) on conflict do nothing;`

You can do it this way in place of hard-coding